src/account.rs

```diff
@@ -92,6 +97,33 @@ pub(crate) struct Balance {
     unverified: Unverified,
 }
 
+#[derive(Debug, Args)]
+pub(crate) struct Transfer {
+    // Bech32Address of the account to transfer assets to.
+    #[clap(long)]
+    to: Bech32Address,
+    // Amount (in u64) of assets to transfer.
+    #[clap(long)]
+    amount: u64,
+    // Asset ID of the asset to transfer.
```

Comment:
```suggestion
    /// Asset ID of the asset to transfer.
```

Reply:
> Looks really exciting! I will test this out first thing tomorrow and approve if it is still around but left some minor things. Not sure if they are intentional though, feel free to discard.

damn, missed the exact things i reviewed on your PR too :laughing: